browser-test/test.grpc.ts

```diff
@@ -0,0 +1,99 @@
+import * as assert from 'assert';
+import {expect} from 'chai';
+
+import * as protobuf from 'protobufjs';
+import { GrpcClient } from '../src/browser';
+import * as gax from '../src/gax';
+
+import { echoProtoJson } from './fixtures/keys';
+
+
+const authStub = {
+    getRequestHeaders() {
+        return { "Authorization": "Bearer SOME_TOKEN" };
+    }
+};
+
+const opts = {
+    auth: authStub,
+};
+
+const gaxGrpc = new GrpcClient(opts);
+
+describe('loadProto', () => {
+
+    it('should create a root object', () => {
+        const protos = gaxGrpc.loadProto(echoProtoJson);
+
+        assert(protos instanceof protobuf.Root);
+        assert(protos.lookupService('Echo') instanceof protobuf.Service);
+        assert(protos.lookupType('EchoRequest') instanceof protobuf.Type);
+    });
+
+    it('should be able to load no files', () => {
+        const protos = gaxGrpc.loadProto({});
+        assert(protos instanceof protobuf.Root);
+
+        assert(protos.nested === undefined);
+        assert.strictEqual(protos.nested, undefined)
+    });
+
+});
+
+describe('createStub', () => {
+
+    let protos, echoService, stubOptions, stubExtraOptions;
+    
+    beforeEach(() => {
+      protos = gaxGrpc.loadProto(echoProtoJson);
+      echoService = protos.lookupService('Echo');
+      stubOptions = {
+        servicePath: 'foo.example.com',
+        port: 443,
+       };
+       stubExtraOptions = {
+        servicePath: 'foo.example.com',
+        port: 443,
+        'grpc.max_send_message_length': 10 * 1024 * 1024,
```

Comment:
we won't need this `grpc.` options, let's remove them (so that we don't create an impression that we support them :) )